### DISCUSSION TITLE
HTTP request to start video capture

### DISCUSSION THREAD

**Discussion by gerryday** [gopro/OpenGoPro#215] [Q&A] [answered]
Hello, sorry if the answer is obvious but I am new to this. Is there a HTTP 2.0 command that would start/stop video recording? I'm using a GoPro Hero 10 Black and the USB connection. I tried the webcam, stream modes, starting video mode and setting the shutter on/off  but the recording doesn't save to the camera.  Thank you

**Comment by gerryday** [comment, marked as answer]
Well I found the solution, I tried to remove the discussion but couldn't figure out how to. Anyway I just needed to enable the usb control from the endpoint. I was able to capture video with the following endpoints:
/gopro/camera/control/wired_usb?p=1
/gopro/camera/shutter/start
/gopro/camera/shutter/stop

**Comment by jaydenlawson** [comment]
Gerry, I'd love to hear how you learned to go from no understanding of the USB control, to at least achieving start/stop control. Would you mind sharing either your code, or the resource that you used to learn how to do this?  I'll be using Python on Ubuntu.  All the examples that I can find are referring to wireless control, [and it seems that I'm not alone](https://stackoverflow.com/questions/72165962/connect-to-gopro-through-usb-with-opengopro).  There must be many more people in the same situation as I am in.
